### ISSUE TITLE
Create courses with only one role

### ISSUE THREAD

**Issue by tv2-pvin** [anthiras/crimson#11]
As an instructor

I would like the ability to create courses for only one role

**Comment by anthiras** [comment]
Good idea, or maybe just no roles. Then it would also support classes where everyone can participate where there are no roles.

**Comment by tv2-pvin** [comment]
well actually both one role and no roles. We occasionally have classes for only follows and for only leads, as well as classes where no role is required.
